### PR TITLE
Always is a bold claim

### PR DESCRIPTION
**Pull request by mgifford**
Much better to say that this is something that we strive for.

[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/mgifford-patch-5/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/05-engineering/accessibility.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ We implement 508 and WCAG compliant websites so that people with all types of di
 
 ## When we do this
 
-- We always produce work that is accessible to people of all abilities, regardless of client. However, we recognize that the level of accessibility compliance and prioritization can be influenced by budgetary and contractual implications.
+- We strive produce work that is accessible to people of all abilities, regardless of client. However, we recognize that the level of accessibility compliance and prioritization can be influenced by budgetary and contractual implications.
```
Comment (Tom-Camp) [inline docs/05-engineering/accessibility.md:16]: Agreed, but you should change that to *strive _to_*.

Comment (mgifford) [reply docs/05-engineering/accessibility.md:16]: Good catch. So do I revise the simple patch or just wait for the reviewer to modify it?

Comment (grugnog) [reply docs/05-engineering/accessibility.md:16]: @mgifford I think you can add a commit to the PR branch to address this (anyone on the team can review most handbook changes, except for policy/legal stuff)

**Comment by mgifford** [comment]
Easier to do a new PR than amend this one.

**Pull request closed without merging**